frontend/ui/lib/widgets/routing_button_widget.dart

```diff
@@ -2,18 +2,21 @@ import 'package:flutter/material.dart';
 
 // A Widget for making a linkable button
 class RoutingButton extends StatelessWidget {
+  final String description;
   final String route;
   final Icon iconType;
 
-  RoutingButton(this.route, this.iconType);
+  RoutingButton(this.route, this.iconType, this.description);
 
   @override
   Widget build(BuildContext context) {
-    return RaisedButton(
-      onPressed: () {
-        Navigator.pushNamed(context, route);
-      },
-      child: iconType,
-    );
+    return Tooltip(
```

Comment:
The tooltip shouldn’t be it’s own widget. Not sure if this was pulled in from a different change or if it’s new in this one.